src/components/Board.tsx

```diff
@@ -0,0 +1,18 @@
+//Board component
+import React, { useState, useEffect } from 'react';
+import { Card, Cardback, Hand } from 'react-deck-o-cards';
+import { defHandStyle } from '../constants';
+
+const Board = (props) => {
```

Comment:
Make an interface for your props in all your functional components that take props in